packages/admin-ui/client/pages/List/NoResults.js

```diff
@@ -0,0 +1,69 @@
+/** @jsx jsx */
+
+import { jsx } from '@emotion/core';
+
+import { Button } from '@arch-ui/button';
+import { InfoIcon } from '@arch-ui/icons';
+import { colors } from '@arch-ui/theme';
+
+import { useListPagination } from './dataHooks';
+
+const NoResultsWrapper = ({ children, ...props }) => (
+  <div
+    css={{
+      alignItems: 'center',
+      color: colors.N30,
+      display: 'flex',
+      flexDirection: 'column',
+      fontSize: 32,
+      justifyContent: 'center',
+      padding: '1em',
+      textAlign: 'center',
+    }}
+    {...props}
+  >
+    <InfoIcon css={{ height: 48, width: 48, marginBottom: '0.5em' }} />
+    {children}
+  </div>
+);
+
+export const NoResults = ({ currentPage, filters, itemCount, list, search }) => {
+  const { onReset } = useListPagination(list.key);
```

Comment:
I just saw above this that there's `useReset` which returns `onReset` as well(i assume). I'm not a fan of having a bunch of different ways to do the exact same thing because it raises questions like "why did the author use this hook over here and that hook over there?" and "what hook should i use for this new code?"

Reply:
That's fair enough. The `useReset` hook returns a function that resets all search, filters, columns, sort.

I'll remove `onReset` from pagination --> folks can just write their own from `onChange`.